prisma/data.mjs

```diff
@@ -569,3 +569,21 @@ export const psychologyMethods = [
     title: 'Інше',
   },
 ];
+
+export const donationDetails = {
+  donationEnabled: true,
```

Comment:
This is not a strict rule, but the naming of Boolean variables should start with `is...`, `isDonationEnabled `

Reply:
Thanks for comment! Renamed.